src/nvim/lua/executor.c

```diff
@@ -1220,6 +1221,7 @@ void ex_luado(exarg_T *const eap)
       break;
     }
     if (lua_isstring(lstate, -1)) {
+      size_t old_line_len = STRLEN(ml_get_buf(curbuf, l, false));
```

Comment:
use a variable for `ml_get_buf(curbuf, l, false))`

Reply:
Ah, I see what you mean now - I didn't see the previous call to `ml_get_buf` before.